pkg/db/db.go

```diff
@@ -58,7 +57,9 @@ type Client interface {
 
 	GetDock(dckID string) (*model.DockSpec, error)
 
-	ListDocks() ([]*model.DockSpec, error)
+	GetDocks() ([]*model.DockSpec, error)
```

Comment:
Can you keep ListDocks method the same with before, and add a new method(for example `ListDocksWithFilter` or `ListFilteredDocks` such stuffs), in this way it should reduce some code changes.